lib/msf/core/payload_set.rb

```diff
@@ -183,7 +179,6 @@ def recalculate
         new_keys.push combined
 
         # Cache the payload's size
```

Comment:
Looks like this is a dead comment now 👀 

```suggestion

        # Cache the payload's size
```

Reply:
Removed